### DISCUSSION TITLE
Fix stupid design decision

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16648] [General]
Instead of the header taking up a sixth of the screen space, why not put the tabs to the right of the close button? I'd also get rid of the ugly blur effects going on across the entire app.

![Screenshot_2023-02-07-06-52-44-60_65ad86f86fd2321a85272a99323da10f](https://user-images.githubusercontent.com/65834069/217250243-d4dbbd50-7be0-4ba0-9f04-004e6dcfbfe3.jpg)

**Comment by flexsurfer** [comment]
cc @pedro-et

**Comment by du82** [comment]
The color on this page is also wrong now

![Screenshot_2023-02-07-07-01-03-52_65ad86f86fd2321a85272a99323da10f](https://user-images.githubusercontent.com/65834069/217251888-21fd2b1c-beaa-4891-bdc9-0321d02f4102.jpg)
